src/test/java/com/team3/central/EmailServiceTest.java

```diff
@@ -0,0 +1,18 @@
+package com.team3.central;
+
+import com.team3.central.services.EmailServiceImpl;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class EmailServiceTest {
+    @Autowired
+    EmailServiceImpl emailService;
+
+    @Test
+    void sendEmailTest() {
+        emailService.sendSimpleMessage("io2testmail@gmail.com", "test", "test content");
+    }
```

Comment:
Test wouldn't be like this, but I understand that it is only for demo purposes and will be changed later in dev

Reply:
Exactly